src/libawkward/Identities.cpp

```diff
@@ -17,6 +17,7 @@
 #include "awkward/Identities.h"
 
 namespace awkward {
+  void printMe();
```

Comment:
this function must be a method of `Identities` class. `Identities` class methods are defined in` Identities.h`.